security/oauth/manager/config.go

```diff
@@ -7,17 +7,13 @@ import (
 	"golang.org/x/oauth2/clientcredentials"
 )
 
-type Endpoint struct {
-	TokenURL string `envconfig:"TOKEN_URL" env:"TOKEN_URL"`
-}
-
 type Config struct {
-	ClientID       string        `envconfig:"CLIENT_ID" env:"CLIENT_ID"`
-	ClientSecret   string        `envconfig:"CLIENT_SECRET" env:"CLIENT_SECRET"`
-	Scopes         []string      `envconfig:"SCOPES" env:"SCOPES"`
-	Endpoint       Endpoint      `envconfig:"ENDPOINT" env:"ENDPOINT"`
-	Audience       string        `envconfig:"AUDIENCE" env:"AUDIENCE"`
-	RequestTimeout time.Duration `envconfig:"REQUEST_TIMEOUT" env:"REQUEST_TIMEOUT" default:"10s"`
+	ClientID       string        `yaml:"clientID" json:"clientID"`
+	ClientSecret   string        `yaml:"clientSecret" json:"clientSecret"`
+	Scopes         []string      `yaml:"scopes" json:"scopes"`
+	TokenURL       string        `yaml:"tokenUrl" json:"tokenUrl"`
```

Comment:
please use for YAML token*URL* key to keep shortnames capital as with did with clientID.